### PR TITLE
Elijah/datetime helper

### PR DESCRIPTION
**Pull request by ElijahCaine**
Note: I branched off of the tschuy/minor_test_refactor. Once that gets merged this should look more reasonable. I will also do a rebase after that happens.

Once the changes from  tschuy/minor_test_refactor are merged there will be less commits. As it stands the only changes happened in ``src/helpers.js`` and ``tests/helpers.js``.

### CODE REVIEW

File: tests/helpers.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        });
+
+        it('Returns Unix Timestamp for valid dateTime', function() {
+            expect(helper.checkDateTime('2015-04-28 16:30')
+                         .to.equal(1430264400000));
+        });
+
+        it('Returns `false` for a non-valid date', function() {
+            expect(helper.checkDateTime('2015-04-82')
+                         .to.equal(false));
```
Comment (tschuy) [inline tests/helpers.js]: I think undefined might be a better return? It's still falsey so we can still treat it essentially like false.

File: tests/helpers.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+module.exports = function(expect) {
+    describe('checkDateTime helper function', function() {
+        it('Returns Unix Timestamp for valid date', function() {
+            expect(helper.checkDateTime('2015-04-28')
+                         .to.equal(1430179200000));
+        });
+
+        it('Returns Unix Timestamp for valid dateTime', function() {
+            expect(helper.checkDateTime('2015-04-28 16:30')
+                         .to.equal(1430264400000));
```
Comment (tschuy) [inline tests/helpers.js]: 1430263800000 ?

File: tests/helpers.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+// test/helpers.js
+
+var helper = require('../src/helpers');
+
+module.exports = function(expect) {
+    describe('checkDateTime helper function', function() {
+        it('Returns Unix Timestamp for valid date', function() {
+            expect(helper.checkDateTime('2015-04-28')
+                         .to.equal(1430179200000));
```
Comment (tschuy) [inline tests/helpers.js]: 1430179200 ?

**Comment by iankronquist** [comment]
Maybe I don't quite understand. Are we reimplementing [Date.parse](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse)? It parses a string date in one of several formats and returns either a Date object or undefined. Is that what checkDateTime is supposed to do?

**Comment by tschuy** [comment]
We're writing a wrapper for it. I'm ... not entirely sure why.

**Comment by iankronquist** [comment]
Maybe your time is better spent elsewhere? Luckily most of this can be reused in other helper functions.

**Comment by ElijahCaine** [comment]
@tschuy @iankronquist Do you think just using Date.parse instead of wrapping it is better? This was just on the todo list. The actual function checkDateTime is a wrapper for Date.parse.

**Comment by iankronquist** [comment]
I think we should just use Date.parse. It does what we need. We don't need to spend time testing, reviewing, and maintaining it.

**Comment by tschuy** [comment]
yeah, personally I'd be in favor of scrapping this

**Comment by ElijahCaine** [comment]
functionality is not necessary. Closing PR

**Pull request closed without merging**